ui/apps/platform/cypress/integration/vulnerabilities/exceptionManagement/pendingRequestsTable.test.js

```diff
@@ -0,0 +1,121 @@
+import withAuth from '../../../helpers/basicAuth';
+import { hasFeatureFlag } from '../../../helpers/features';
+import {
+    cancelAllCveExceptions,
+    fillAndSubmitExceptionForm,
+    selectSingleCveForException,
+    verifyExceptionConfirmationDetails,
+    verifySelectedCvesInModal,
+    visitWorkloadCveOverview,
+} from '../workloadCves/WorkloadCves.helpers';
+import { visitExceptionManagement } from './ExceptionManagement.helpers';
+
+describe('Exception Management Pending Requests Page', () => {
+    withAuth();
+
+    before(function () {
+        if (
+            !hasFeatureFlag('ROX_VULN_MGMT_WORKLOAD_CVES') ||
+            !hasFeatureFlag('ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL') ||
+            !hasFeatureFlag('ROX_WORKLOAD_CVES_FIXABILITY_FILTERS')
+        ) {
+            this.skip();
+        }
+    });
+
+    beforeEach(() => {
+        if (
+            hasFeatureFlag('ROX_VULN_MGMT_WORKLOAD_CVES') &&
+            hasFeatureFlag('ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL') &&
+            hasFeatureFlag('ROX_WORKLOAD_CVES_FIXABILITY_FILTERS')
+        ) {
+            cancelAllCveExceptions();
+        }
+    });
+
+    after(() => {
+        if (
+            hasFeatureFlag('ROX_VULN_MGMT_WORKLOAD_CVES') &&
+            hasFeatureFlag('ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL') &&
+            hasFeatureFlag('ROX_WORKLOAD_CVES_FIXABILITY_FILTERS')
+        ) {
+            cancelAllCveExceptions();
+        }
+    });
+
+    it.skip('should be able to view deferred pending requests', () => {
+        visitWorkloadCveOverview();
```

Comment:
One thing we should consider here for the future is that it might be safer to ensure we "clear filters" after the initial visit to the Workload CVE page.

There have been issues lately where there is not enough data, or no data at all, when visiting the page which causes test flakes. Removing the applied default filters could help mitigate this issue by allowing all CVEs to be present.

Reply:
Ah ok, I see you do that in other places. Is it better to include this within the `visitWorkloadCVEOverview` function to prevent future discrepancies in tests?